chart/templates/rbac.yaml

```diff
@@ -0,0 +1,439 @@
+{{- if .Values.rbac.create -}}
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: {{ include "tekton-operator.serviceAccountName" . }}
+  labels:
+    {{- include "tekton-operator.labels" . | nindent 4 }}
+  {{- with .Values.rbac.serviceAccount.annotations }}
+  annotations:
+    {{- toYaml . | nindent 4 }}
+  {{- end }}
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  name: {{ include "tekton-operator.fullname" . }}
+  labels:
+    {{- include "tekton-operator.labels" . | nindent 4 }}
+rules:
+  - apiGroups:
+      - ""
+    resources:
+      - services
+      - endpoints
+      - pods
+    verbs:
+      - get
+      - list
+      - watch
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: {{ include "tekton-operator.fullname" . }}
+  labels:
+    {{- include "tekton-operator.labels" . | nindent 4 }}
+rules:
```

Comment:
this rules are for service account for kubernetes platform

for openshift the rules are bit different
https://github.com/tektoncd/operator/blob/main/config/openshift/base/role.yaml

so currently on openshift helm chart is failing